app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -35,6 +35,7 @@
       android:layout_marginEnd="@dimen/spacing_8"
       android:text="@string/patientsummary_edit"
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button2.White100"
+      android:visibility="invisible"
```

Comment:
We should override the visibility in the editor so that it's visible in the preview?

Reply:
Nice catch.